### DISCUSSION TITLE
How can I do the accounting currency sign (eg) brackets around num for negative vals

### DISCUSSION THREAD

**Discussion by JackEdwardLyons** [dm4t2/vue-currency-input#285] [Ideas]
Is there a way to add brackets around numbers when they are negative instead of the `-` symbol.

For example, when a user presses `-` the number gets automatically wrapped with `( )` symbols.

Eg. `-$2,000 => (2,000)`

Using the `Intl.NumberFormat` there is a `currencySign` option which can be set to `accounting`

Eg. 
```
Intl.NumberFormat('en-US', {
      currencySign: 'accounting',
      currency: 'USD',
      style: 'currency',
      currencyDisplay: 'narrowSymbol',
}).format(num)
```

Thanks!

**Comment by dm4t2** [comment]
Thanks for your suggestion. I'll take a look at it.

**Comment by dm4t2** [comment]
Now available in [v2.4.0](https://github.com/dm4t2/vue-currency-input/releases/tag/2.4.0) 🎉
